spec/src/main/asciidoc/bulkhead.asciidoc

```diff
@@ -71,8 +71,19 @@ public Future<Connection> serviceA() {
 
 When using the thread pool approach, when a request cannot be added to the waiting queue, `BulkheadException` will be thrown.
 
+==== Lifecycle
+
+Unlike some other fault tolerance strategies, bulkhead needs to store some data between invocations: the number of currently running executions, or the queue of waiting executions.
+This state is a singleton, irrespective of the lifecycle of the bean that uses the `@Bulkhead` annotation.
```

Comment:
It is associated with the class instead of the instances.

Reply:
I've changed it to use both explanations.